### PR TITLE
probe: add unit-tests for TFModel struct

### PR DESCRIPTION
**Pull request by thangng48**

### CODE REVIEW

File: probe/tfmodel/tfmodel_test.go [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+			name: "path is not existing",
+			conf: Config{},
+			args: args{
+				ctx:     context.Background(),
+				baseDir: "somewhere",
+			},
+			wantErr: true,
+		},
+		{
+			name: "path is existing but not contains model",
```
Comment (maxhora) [inline probe/tfmodel/tfmodel_test.go:86]:
```suggestion
			name: "conf.path doesn't contain model",
```

File: probe/tfmodel/tfmodel_test.go [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+		baseDir string
+	}
+	tests := []struct {
+		name    string
+		conf    Config
+		args    args
+		wantErr bool
+	}{
+		{
+			name: "path is not existing",
```
Comment (maxhora) [inline probe/tfmodel/tfmodel_test.go:77]:
```suggestion
			name: "conf.path is not provided",
```

File: probe/tfmodel/test/archive.go [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+	}
+	testBaseDir string
+)
+
+func init() {
+	_, file, _, _ := runtime.Caller(0)
+	testBaseDir = regexp.MustCompile(`archive\.go`).ReplaceAllString(file, "")
+}
+
+//UnzipModels unzip testing models to temporary dir
```
Comment (maxhora) [inline probe/tfmodel/test/archive.go:40]:
```suggestion
// UnzipModels unzip testing models to temporary dir
```

**Comment by maxhora** [comment]
should we test `func NewModel(info Config) *TFModel` as well there?

**Comment by thangng48** [comment]
> should we test `func NewModel(info Config) *TFModel` as well there?

I changed from literal initiation to call NewModel function for creating `*TFModel`.

**Review by Dicklesworthstone — APPROVED** [review]

**Review by ashadi-cc — APPROVED** [review]

**Pull request merged**